net-dns/ddclient/files/ddclient-3.11.2-fix-automake-portability-warning.patch

```diff
@@ -0,0 +1,41 @@
+https://github.com/ddclient/ddclient/commit/a12398c315b9b909e57e87acf9fd3a15a0b3e213.diff
```

Comment:
Not really an issue, but I prefer using `.patch` instead of `.diff` as it preserves the commit information.

Reply:
Makes sense, I followed your suggestion in the force-push below

Reply:
I meant that I commit the entire file `.patch` file with the upstream commit information included.

Reply:
Of course, I should have understood that was what you meant :) Changed the commit accordingly